src/gax-internal/src/options.rs

```diff
@@ -19,6 +19,19 @@ pub type ClientConfig = gax::client_builder::internal::ClientConfig<Credentials>
 
 pub(crate) const LOGGING_VAR: &str = "GOOGLE_CLOUD_RUST_LOGGING";
 
+/// Information about the client library used for instrumentation.
+#[derive(Clone, Debug)]
```

Comment:
I believe we can also make this `#[derive(Clone, Copy, Debug)]` since it is just a collection of `static` references.  If we think copying may become expensive in some distant future then just `Clone` is safer.